srcpkgs/yaboot/template

```diff
@@ -23,7 +23,12 @@ nostrip_files=yaboot
 # We need to compile yaboot binary itself, as well as e2fsprogs libraries,
 # using a 32-bit ppc compiler; 64-bit with -m32 won't do because of libgcc etc.
 case "$XBPS_TARGET_MACHINE" in
-	ppc64*) _need_cross=powerpc-linux-musl ;;
+	ppc64*) if [ "$XBPS_LIBC" = "musl" ]; then
```

Comment:
why?

Reply:
```
./xbps-src -a ppc64 -t -N pkg yaboot
…
=> yaboot-1.3.17_1: running pre_build ...
/builddir/yaboot-1.3.17/e2fsprogs-1.41.14 /builddir/yaboot-1.3.17
autoreconf: configure.in: AM_GNU_GETTEXT is used, but not AM_GNU_GETTEXT_VERSION
Generating configuration file for e2fsprogs version 1.41.14
Release date is December, 2010
checking build system type... x86_64-unknown-linux-gnu
checking host system type... Invalid configuration `powerpc-linux-musl': system `musl' not recognized
configure: error: /bin/sh config/config.sub powerpc-linux-musl failed
=> ERROR: yaboot-1.3.17_1: pre_build: 'CFLAGS="$_ecflags" CC="$_ecc" AR="$_ear" ./configure ${_args}' exited with 1
=> ERROR:   in _build_e2fsprogs() at srcpkgs/yaboot/template:70
=> ERROR:   in pre_build() at srcpkgs/yaboot/template:98
```

Reply:
that just means you should update config.sub/guess, not this

Reply:
Latest config.{guess,sub} from http://git.savannah.gnu.org/cgit/config.git/tree/ do not solve this.

Reply:
i'll look at it later, but this is definitely not a proper fix, there's some other issue here

Reply:
Removed that hack for now and left just the `-fcommon` addition to `{YB,U}CFLAGS`.